<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MAVBNG1121337" failures="3" tests="4" name="Test" time="13.048" errors="0" timestamp="2022-01-15T08:00:40 IST">
  <testcase classname="ValuesAssign.CalTest" name="sumTest" time="2.199"/>
  <testcase classname="ValuesAssign.CalTest" name="subTest" time="2.046">
    <failure type="java.lang.AssertionError" message="expected [80] but found [120]">
      <![CDATA[java.lang.AssertionError: expected [80] but found [120]
at ValuesAssign.CalTest.subTest(CalTest.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- subTest -->
  <testcase classname="ValuesAssign.CalTest" name="mulTest" time="2.122">
    <failure type="java.lang.AssertionError" message="expected [200] but found [12]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [12]
at ValuesAssign.CalTest.mulTest(CalTest.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- mulTest -->
  <testcase classname="ValuesAssign.CalTest" name="divTest" time="2.02">
    <failure type="java.lang.AssertionError" message="expected [2] but found [30]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [30]
at ValuesAssign.CalTest.divTest(CalTest.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- divTest -->
</testsuite> <!-- Test -->
